lib/chart.js

```diff
@@ -189,19 +188,15 @@ export const findChartSize = (options) => {
   const globalOptions = isCorrectJSON(options.export?.globalOptions);
 
   // Secure scale value
-  let scale = roundNumber(
+  let scale =
     options.export?.scale ||
-      exporting?.scale ||
-      globalOptions?.exporting?.scale ||
-      options.export?.defaultScale ||
-      1
-  );
+    exporting?.scale ||
+    globalOptions?.exporting?.scale ||
+    options.export?.defaultScale ||
+    1;
 
-  if (scale > 5) {
-    scale = 5;
-  } else if (scale < 0.1) {
-    scale = 1;
-  }
+  // the scale cannot be lower than 0.1 and cannot be higher than 5.0
+  scale = Math.max(0.1, Math.min(scale, 5.0));
```

Comment:
I think that we should allow values rounded to 2 decimal places at most. We don't want values such as 0.235123543.